src/Illuminate/Contracts/Database/Events/Migration.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Illuminate\Contracts\Database\Events;
+
+interface Migration
```

Comment:
This should be called `MigrationEvent` btw since it's not a migration but an event.